test/Microsoft.Extensions.CodeGeneration.Core.FunctionalTest/project.json

```diff
@@ -1,27 +1,28 @@
 {
     "dependencies": {
-        "Microsoft.Extensions.PlatformAbstractions": "1.0.0-*",
-        "Microsoft.Extensions.DependencyInjection": "1.0.0-*",
-        "Microsoft.Extensions.CodeGeneration.Core": "1.0.0-*",
-        "Microsoft.AspNetCore.Hosting": "1.0.0-*",
-        "ModelTypesLocatorTestWebApp": "1.0.0",
-        "xunit.runner.aspnet": "2.0.0-aspnet-*"
-    },
+    "Microsoft.Extensions.CodeGeneration.Utils" : "1.0.0-rc2-15995",
+		"Microsoft.Extensions.DependencyInjection": "1.0.0-rc2-15998",
+		"Microsoft.Extensions.CodeGeneration.Core": "1.0.0-*",
+		"Microsoft.AspNetCore.Hosting": "1.0.0-rc2-16270",
+		"ModelTypesLocatorTestWebApp": "1.0.0",
 
-    "commands": {
-        "test": "xunit.runner.aspnet"
-    },
+    "NETStandard.Library": "1.0.0-rc2-23727",
+    "moq.netcore": "4.4.0-beta8",
+    "xunit.runner.console" : "2.1.0",
+    "Microsoft.NETCore.TestHost" : "1.0.0-rc2-23706",
```

Comment:
If the NETCore.TestHost is not added as dependency, the tests randomly complain that it could not load the assembly. 
Did not spend time to figure out who trying to load this, but adding this as a dependency made the issue go away.